roles/create_tables/tasks/create_spec_managedclusters_table.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: create managed_clusters_labels table
+  tags: tables
+  postgresql_table:
+    table: "{{ schema }}.{{ name }}"
+    db: "{{ hoh_db }}"
+    login_host: "{{ db_login_host }}"
+    login_user: "{{ db_login_user }}"
+    login_password: "{{ db_login_password }}"
+    ssl_mode: "{{ db_ssl_mode }}"
+    columns:
+      - leaf_hub_name         varchar({{ cluster_name_length_limit }}) not null default ''
+      - managed_cluster_name  varchar({{ cluster_name_length_limit }}) not null
+      - labels                jsonb not null default '{}'::jsonb  # format: map[string]string
+      - deleted_label_keys    jsonb not null default '[]'::jsonb  # format: []string
+      - updated_at            timestamp not null default now()
+      - version               bigint not null default 0 check (version >= 0) # version field local for each row
+
+- name: create index on updated_at
```

Comment:
same comment as other table.. didn't understand why index on `updated_at` is needed